yarn-project/bb-prover/package.json

```diff
@@ -73,15 +73,18 @@
     "tslib": "^2.4.0"
   },
   "devDependencies": {
+    "@aztec/ethereum": "workspace:^",
     "@jest/globals": "^29.5.0",
     "@types/jest": "^29.5.0",
     "@types/memdown": "^3.0.0",
     "@types/node": "^18.7.23",
     "@types/source-map-support": "^0.5.10",
     "jest": "^29.5.0",
     "jest-mock-extended": "^3.0.3",
+    "solc": "^0.8.26",
```

Comment:
Curious: why wasn't solc needed before?

Reply:
ah it's not needed. I had added a test file in this package but it was just duplicating the existing integration test in e2e so removed this. I'll need to remove this dependency from `bb-prover`